pkgs/development/python-modules/slpp/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "slpp";
+  version = "1.2.3";
```

```suggestion
  version = "1.2.3";
  pyproject = true;
```
Could you please add this parameter?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters